pkg/runit/runit_service.go

```diff
@@ -80,10 +85,6 @@ func (sv *SV) Stop(service *Service) (string, error) {
 	return convertToErr(sv.execOnService(service, "stop"))
```

Comment:
you might want to add the timeout behavior to stop as well? not sure if we actually use this anywhere though